performance-test/src/main/java/org/beiter/michael/eaudit4j/performance/cassandra/CassandraServerUtil.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * #%L
+ * This file is part of eAudit4j, a library for creating pluggable
+ * auditing solutions, providing an application to execute performance tests agains supported databases.
+ * %%
+ * Copyright (C) 2015 - 2016 Michael Beiter <michael@beiter.org>
+ * %%
+ * All rights reserved.
+ * .
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the copyright holder nor the names of the
+ *       contributors may be used to endorse or promote products derived
+ *       from this software without specific prior written permission.
+ * .
+ * .
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+package org.beiter.michael.eaudit4j.performance.cassandra;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.PoolingOptions;
+import com.datastax.driver.core.HostDistance;
+import com.datastax.driver.core.ProtocolVersion;
+import com.datastax.driver.core.Session;
+
+import static org.beiter.michael.eaudit4j.performance.PropertiesUtil.getIntProperty;
+
+public class CassandraServerUtil {
+    private static final String CONTACT_POINTS = "contactPoints";
+    private static final String PORT = "port";
+    private static final String USERNAME = "username";
+    private static final String PASSWORD = "password";
+
+    private static String contactPoints = System.getProperty(CONTACT_POINTS, "192.168.99.100");
```

Comment:
"192.168.99.100" is the default IP from Docker Machine. Before runt the tests, the Cassandra Server must be up and it can be done using the dorcker-compose file present in this PR.
If the SO is Linux, the docker's IP is the localhost then start the application with ```-DcontactPoints=127.0.0.1```